modules/api/current.js

```diff
@@ -431,6 +431,37 @@ module.exports = {
           level: 'ERROR',
           message: stripAnsi(testStepResult.message),
         });
+
+        const hasBrowser = 'browser' in global;
+        const isTakeScreenshotOptionProvidedInRPConfig =
+          this.config.takeScreenshot && this.config.takeScreenshot === 'onFailure';
```

Comment:
Please update the changelog and add corresponding option to [readme](https://github.com/reportportal/agent-js-cucumber/tree/master/modules/api/deprecated#install-agent-to-your-project-dir) as for deprecated version.